content/en/docs/languages/ruby/basics.md

```diff
@@ -334,6 +334,31 @@ resps.each do |r|
 end
 ```
 
+Streaming RPCs block until the server closes the connection. For non-blocking
+operation, multiple threads must be utilizes as well as the `return_op: true`
+flag. The flag changes the response of the call from a blocking enumerable to a
```

Comment:
This might be a bit misleading, since both `return_op: true` and `return_op: false` ultimately provide a blocking enumerable, at least for reading responses from the peer.

The main thing the `Operation` return type provides it is defers execution of (the start of) the RPC until `execute` is called. And then it also provides some more contextual methods and getters like `cancel` and `status`.